sources/packages/web/src/components/aest/students/modals/ApproveApplicationOfferingChangeRequestModal.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <modal-dialog-base :showDialog="showDialog" title="StudentAid BC Declaration">
+    <template #content>
+      <p>
+        Please review and accept the StudentAid BC declaration to allow the
+        institution to make proposed changes to your application (on your
+        behalf).
+      </p>
+      <content-group-info class="my-3">
+        <v-row
+          ><v-col
+            ><h4 class="label-bold color-blue">
```

Comment:
Can you pls review the styles with figma, example line 12 is not a h4 in figma, 
![image](https://github.com/bcgov/SIMS/assets/77353155/8d00943b-a1eb-450d-957b-fc71ce6ced0e)

Reply:
@ann-aot In the screenshot you have shared, it says headline small and that is why I used `h4` tag and it has a `font-size` of 16 px. Keeping it as is but let me know if I missed something else.

Reply:
@sh16011993 normally the Figma will mention if any header tags need to be used, for example, 
![image](https://github.com/bcgov/SIMS/assets/77353155/5ed900b6-e54c-40a9-ad2d-f651ec033b32)

Reply:
@ann-aot Since the figma mentions as **Headline small**, keeping it as is. class `label-bold` applies the topography `font-size`, `line-height` and `font-weight`.